src/main.tsx

```diff
@@ -18,9 +19,14 @@ const router = createBrowserRouter([
   },
 ]);
 
+// Preload any cached theme variables to avoid a flash of unstyled content
+preloadTheme();
+
 ReactDOM.render(
   <React.StrictMode>
-    <RouterProvider router={router} />
+    <ThemeProvider themes={null}>
```

Comment:
Setting `themes={null}` causes children not to be rendered since it waits for active themes to load. Looks like it will need to be set to an empty array.

Reply:
See
* https://github.com/deephaven/web-client-ui/blob/main/packages/components/src/theme/ThemeProvider.tsx#L59
* https://github.com/deephaven/web-client-ui/blob/main/packages/components/src/theme/ThemeProvider.tsx#L113